source/projects/nanogen.md

```diff
@@ -0,0 +1,24 @@
+---
+title: NanoGen
+repo: doug2k1/nanogen
+homepage: https://github.com/doug2k1/nanogen
+language: JavaScript
+license: MIT
+templates: EJS, Markdown
```

Comment:
"Markdown" can be removed here as it's a data format rather than a template format.